en/docs/chapter_searching/searching_algorithm_revisited.md

```diff
@@ -1,48 +1,48 @@
 # Search algorithms revisited
 
-<u>Searching algorithms (searching algorithm)</u> are used to search for one or several elements that meet specific criteria in data structures such as arrays, linked lists, trees, or graphs.
+<u>Searching algorithms (search algorithms)</u> are used to retrieve one or more elements that meet specific criteria within data structures such as arrays, linked lists, trees, or graphs.
 
```

Comment:
- Keep consistency with the title - "Search algorithms revisited"
- We search for an element but then we need to do something with it - "retrieve"
- the "specific criteria" relates to the element, not the data structure - add 'within' to make this clearer